### ISSUE TITLE
v5.0.0 Model\SeenAt issues

### ISSUE THREAD

**Issue by SergkeiM** [fingerprintjs/fingerprint-pro-server-api-php-sdk#123]
Hi,

After upgrading to `v5` I get this error:

```php
Fingerprint\ServerAPI\Model\SeenAt::getGlobal(): Return value must be of type DateTime, null returned
```
`seen_at` - can be nullable, but `Model` doesn't accepts nullable value.

**Comment by ilfa** [comment]
Hi @SergkeiM!
Sorry for the late answer and thank you for the issue.
We are going to fix this problem soon.

**Comment by ilfa** [comment]
It should be fixed now in [v5.1.1](https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/releases/tag/v5.1.1)
